src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,268 @@
+import React, {
+  FC,
+  ReactNode,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react'
+import styled, { css } from 'styled-components'
+import { Text } from '../Text'
+import { theme } from '../theme'
+
+type Position = 'top' | 'bottom' | 'left' | 'right'
+type Size = 'small' | 'large'
+type ArrowPosition = 'left' | 'right' | 'center' | 'top' | 'bottom'
+
+export interface TooltipProps {
+  children: ReactNode
+  title?: string
+  content: string | ReactNode
+  position: Position
+  underline?: boolean
+  size?: Size
```

Comment:
can't we derive the `size` from the content and maybe have just a fixed `max-width`? In the Figma file doesn't seem like there's specified any specific values for the size

Reply:
Ahh, you're right, I was running off the M5 Figma and not the design systems. Will add a max width of `240px` (largest tooltip used on the M5 design).

Reply:
Sweet!